components/site/src/lib.rs

```diff
@@ -69,7 +69,7 @@ impl Site {
         config.load_extra_syntaxes(path)?;
 
         let tpl_glob =
-            format!("{}/{}", path.to_string_lossy().replace("\\", "/"), "templates/**/*.*ml");
+            format!("{}/{}", path.to_string_lossy().replace("\\", "/"), "templates/**/*");
```

Comment:
That's going to mess up with various editor swap files..

Reply:
Oh, right. I have vim configured with `directory=~/.local/share/nvim/swap//` so 
I didn't notice that. If this is a bad problem, the target file extension needs to be set in front matter. Or alternatively there needs to be another way of selecting files, eg. multiple globwalk patterns or a regex, but that would require changes to Tera.